examples/async_transcribe_media_from_url/async_transcribe_media_from_url.js

```diff
@@ -0,0 +1,60 @@
+const revai = require('revai-node-sdk');
+const fs = require('fs');
+const token = require('../config/config.json').access_token;
+
+(async () => {  
+    /* Initialize your client with your revai access token */
+    var client = new revai.RevAiApiClient(token);
+
+    /* Get account details */
+    var account = await client.getAccount();
+    console.log(`Account: ${account.email}`);
+    console.log(`Balance: ${account.balance_seconds} seconds`);
+
+    const jobOptions = {
+        metadata: "InternalOrderNumber=123456789",
+        callback_url: "https://jsonplaceholder.typicode.com/posts",
+        skip_diarization: true,
+        custom_vocabularies: [{
+            phrases: [
+                "add",
+                "custom",
+                "vocabularies",
+                "here"
+                ]
+            }]
+    };
+
+    /* Media may be submitted from a url */
+    var job = await client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3", jobOptions);
+
+    console.log(`Job Id: ${job.id}`);
+    console.log(`Status: ${job.status}`);
+    console.log(`Created On: ${job.created_on}`);
+
+    // Waits 5 seconds between each status check to see if job is complete
```

Comment:
should we add a `note: polling for job status is not recommended in a non-testing environment. Use the callback_url option [link to documentation] to receive the response asynchronously on job completion`?

Reply:
This shows as outdated but has not been addressed @menioa please do this